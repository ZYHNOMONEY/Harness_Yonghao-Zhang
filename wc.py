def word_count(filename):
    
    try:
        with open(filename, 'r') as file:
            lines = file.readlines()

        num_lines = len(lines)
        num_words = sum(len(line.split()) for line in lines)
        num_chars = sum(len(line) for line in lines)

        return num_lines, num_words, num_chars
    except FileNotFoundError:
        return "File not found."