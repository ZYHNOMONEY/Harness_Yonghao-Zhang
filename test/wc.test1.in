test file
1

finish