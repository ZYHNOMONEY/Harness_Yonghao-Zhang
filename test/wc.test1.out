      4      4     19 test/wc.test1.in